### ISSUE TITLE
SIDD and DND benchmark code

### ISSUE THREAD

**Issue by dgjung0220** [xuan611/sRGB-Real-Noise-Synthesizing#5]
Hello. The NeCA model has been trained separately for each device type, with distinct generator and denoiser models for each. However, there is no information about device types in the SIDD and DND benchmarks, and especially in the case of DND, the device types are not consistent. How were the experimental results in Table 2 conducted? I'm inquiring whether you can provide the relevant code.

**Comment by zfu006** [comment]
Hi, as we wrote in the paper, we use the synthetic SIDD to train the denoiser. For example, if an image is from IP, then you can use the pretrained IP noise generator to synthesize fake noisy image. You can process all the images in SIDD following that process. Finally, you will get a synthetic SIDD dataset. The denoiser shall be trained with this synthetic dataset and tested on the SIDD and DND benchmark. Hoping this answer solves your concerns.

**Comment by dgjung0220** [comment]
I understand. However, I have some curious about the performance results presented in the paper. The performance result for Real noise, 37.12, is higher than Flow-Srgb (36.51) and C2N (35.13). When I followed the training method outlined in the paper (300 epochs, learning rate 1e-3, using Adam, etc.), I achieved a real noise performance of around 36.7 with DnCNN. 
Do you have any plans to provide pretrained denoiser weights for Table 2?

Furthermore, the average of the results presented in Table 1 does not seem to match the table. Is there something I might be missing?
![image](https://github.com/xuan611/sRGB-Real-Noise-Synthesizing/assets/16290330/fbc197ec-8b50-4f6d-879b-8481b71e00ee)
When checking the averages in Excel, it appears as follows.
<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/dgjun/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/dgjun/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">



  |   | AWGN | C2N | NeCA | NLF-ISP | Real
-- | -- | -- | -- | -- | -- | --
G4 | KL | 1.9755 | 0.166 | 0.0242 | 0.0102 | -
  | PSNR | 28.15 | 37.81 | 38.85 | 38.51 | 40.6
GP | KL | 1.8351 | 0.1315 | 0.0432 | 0.0126 | -
  | PSNR | 28.45 | 37.08 | 37.72 | 37.74 | 38.33
IP | KL | 1.8562 | 0.0581 | 0.041 | 0.0475 | -
  | PSNR | 28.01 | 39.12 | 39.46 | 39.53 | 39.45
N6 | KL | 2.1465 | 0.3524 | 0.0206 | 0.0063 | -
  | PSNR | 26.31 | 33.59 | 35.54 | 34.84 | 35.56
S6 | KL | 0.4517 | 0.4517 | 0.0302 | 0.0902 | -
  | PSNR | 27.22 | 33.18 | 35.56 | 35.99 | 36.85
Average | KL | 1.653 | 0.23194 | 0.03184 | 0.03336 | -
  | PSNR | 27.628 | 36.156 | 37.426 | 37.322 | 38.158



</body>

</html>

**Comment by zfu006** [comment]
I suggest you to train the DnCNN with more epochs, say 5000 epochs or more. You can decrease the learning rate to 1e-4 after 500.
